lib/datadog/appsec.rb

```diff
@@ -8,6 +8,24 @@ module Datadog
   module AppSec
     include Configuration
 
+    class << self
+      def enabled?
+        Datadog.configuration.appsec.enabled
+      end
+
+      def processor
+        appsec_component = components.appsec
+
+        appsec_component.processor if appsec_component
+      end
```

Comment:
Following from https://github.com/DataDog/dd-trace-rb/pull/2632#discussion_r1114143568 -- I guess do quickly doublecheck if `#processor` should be considered public or not.

Reply:
We use it here 
https://github.com/DataDog/dd-trace-rb/blob/99052d9ec16b3a9505a25807bdd201b5f3b0dd58/lib/datadog/appsec/contrib/rack/request_middleware.rb#L28

Reply:
We actually have avoided still having these as "Ruby public" in a few places in the codebase -- you'll find a few `.send(...)` if you look for them, which is our attempt at prioritizing the customer communication part ("you shouldn't rely on this") vs the library ergonomics part ("send(...) is really ugly").

But... I don't have strong feelings on this, we're definitely not consistent in our approach here.